arangod/CMakeLists.txt

```diff
@@ -961,6 +961,11 @@ endif ()
 if (USE_JEMALLOC)
   add_dependencies(arangod jemalloc)
 endif ()
+  
+if (USE_MAINTAINER_MODE)
+  add_dependencies(arangod errorfiles)
+  add_dependencies(arangod exitcodefiles)
+endif ()
```

Comment:
While the error- and exitcodefiles are built now, they might be built after `arango` is built, which contains them. So I think it'd be better to move this to `lib/CMakeLists.txt` and as a dependency to `arango`, not `arangod`.

Reply:
True